common/service/dynamicconfig/constants.go

```diff
@@ -257,6 +257,8 @@ var keys = map[Key]string{
 	MutableStateChecksumVerifyProbability:                  "history.mutableStateChecksumVerifyProbability",
 	MutableStateChecksumInvalidateBefore:                   "history.mutableStateChecksumInvalidateBefore",
 	ReplicationEventsFromCurrentCluster:                    "history.ReplicationEventsFromCurrentCluster",
+	FailoverMarkerHeartbeatInterval:                        "history.FailoverMarkerHeartbeatInterval",
+	FailoverMarkerHeartbeatTimerJitterCoefficient:          "history.FailoverMarkerHeartbeatTimerJitterCoefficient",
```

Comment:
nit: also update the naming to `notifyMarkerXXX`?

Reply:
updated